### ISSUE TITLE
Personal contact info?

### ISSUE THREAD

**Issue by hanlonmt** [agaveplatform/pyprofiles-api#1]
Hi, I got a notification that my personal contact info was available in your database. This seems sketchy. I searched and found it. Why is this here? How can it be removed? Can i just fork the repo and take it out?

**Comment by deardooley** [comment]
I’m happy to prune out any user date you identified. I believe I can just remove the db from the git history. This repo is largely archived and the project has been forked and maintained by tacc since 2018, so I’d check their repo as well.
—
Rion

On Nov 27, 2024, at 1:02 PM, Molly Hanlon ***@***.***> wrote:



Hi, I got a notification that my personal contact info was available in your database. This seems sketchy. I searched and found it. Why is this here? How can it be removed? Can i just fork the repo and take it out?

**Comment by hanlonmt** [comment]
That would be great, thanks. It seems like both my given and nickname end
up in here with some associated phone numbers and emails - Meredith (block
on line 6761) and Molly (line 2447)

**Comment by hanlonmt** [comment]
That was in ldap03. It's slightly different in ldap01, but really, anything
that's me would be great to have out of there.

**Comment by TKDBB84** [comment]
Yeah, same my personal info including phone number is dumped here.  You really should delete phone numbers, passwords, emails, full names, what was the source for the ldap data anyway?

**Comment by deardooley** [comment]
My apologies. From the commit history, it looks like the repo was cloned from https://bitbucket.org/tacc-cic/agave_id/src/django1.11/. The databases dumps seem to have been pushed up in the initial commit. The ldap diff script, seems to indicate that the dumps are from TACC's ldap around 2017 or 2018. 

Regardless of the source, I have pruned the ldap0X files from the git history on this repo. I don't have any influence on the original repo. You may have luck reaching out to TACC directly through their website. Again, I apologies for an inconvenience this my have caused you. Hopefully this is now resolved with respect to this repo.

**Comment by TKDBB84** [comment]
Thank you, history looks clean now.


I've also opened a ticket with BitBucket to hopefully get the info removed there as well, as there doesn't seem to be a way to directly get in contact on that site.
